src/spikeinterface/core/base.py

```diff
@@ -798,7 +799,10 @@ def load(file_path: Union[str, Path], base_folder: Optional[Union[Path, str, boo
             return extractor
 
         else:
-            raise ValueError("spikeinterface.Base.load() file_path must be an existing folder or file")
+            error_msg = f"Failed to load the file_path {file_path} must be an existing folder or file"
+            if base_folder:
+                error_msg = f"\n base folder provided is {base_folder}"
```

Comment:
```suggestion
            error_msg = f"Failed to load the file_path {file_path}: it must be an existing folder or file"
            if base_folder:
                error_msg = f"\nThe provided base_folder provided is: {base_folder}"
```

Reply:
Your suggestion repeats provided but yes, my grammar is weird on that one. Let me fix it and push.